src/main/kotlin/com/github/ast/parser/TornadoFX.kt

```diff
@@ -17,9 +24,114 @@ enum class NODES {
     CheckBox, Paginator, PasswordField, TreeView, TabView
 }
 
-// detected inputs to test, will build over time
+/**
+ * UINode Inputs to interact with for testing. Will add over time.
+ */
 enum class INPUTS {
     TextField, Button, Form
 }
 
-data class TornadoFXView(var view: String? = "", var type: String? = "", var scope: String? = "")
\ No newline at end of file
+/**
+ * TornadoFX specific:
+ *    Component breakdown meant to be saved and used for test generation
+ */
+fun saveComponentBreakdown(superClass: String, className: String, parser: KParser) {
```

Comment:
If I'm understanding this correctly, this is kinda neat! Looks like you're able to pass this higher-order function in to specify that we only want to select TFX classes, excluding others.

NIT: Could be written as:

```suggestion
fun saveComponentBreakdown(superClass: String, className: String, parser: KParser) {
    val isTornadoFXView = listOf(COMPONENTS.Fragment.name, COMPONENTS.View.name)
            .contains(superClass)
    if (isTornadoFXView) parser.views[className] = TornadoFXView(className, superClass)
}
```